File: odd_even.java
// Java program program to
// check for even or odd
 
class GFG
{
    // Returns true if n
    // is even, else odd
    public static boolean isEven(int n)
    {
        if((n & 1) == 0)
            return true;
        else
            return false;
    }
     
    // Driver code
    public static void main(String[] args)
    {
        int n = 101;
        if(isEven(n) == true)
            System.out.print("Even");
        else
            System.out.print("Odd");
    }
}

File: odd_even.js
<script>
// A simple JavaScript program to
// check for even or odd
 
// Returns true if n is
// even, else odd
function isEven(n)
{
     
    // n & 1 is 1, then
    // odd, else even
    return (!(n & 1));
}
 
// Driver code
let n = 101;
isEven(n)? document.write("Even") :
        document.write("Odd");

</script>
